views/development/all-templates.njk

```diff
@@ -87,4 +87,6 @@
       text: "Go"
     }) }}
   </form>
+
+  <a class="govuk-link" href="/dev/template/service-unavailable">Service Unavailable static page</a>
```

Comment:
Nice 👍 

I did find where this page is actually being referenced - https://github.com/govuk-one-login/ipv-core-common-infra/blob/7a692a8c0f4125713fa83a94227c2ad20fe7534f/cloudformation/shield-monitor/template.yaml#L19 which is used on DNS failover. But the url is very out of date 🤦